### PR TITLE
MoM-2024-03-12.md

### PR DESCRIPTION
**Pull request by jlurien**
#### What type of PR is this?

* documentation

#### What this PR does / why we need it:

Agenda for next meeting, to be updated with minutes after meeting

### CODE REVIEW

**Review by bigludo7 — COMMENTED** [review]
Minutes updated after the meeting.
@jlurien & team please take a look and will merge once OK.
Thanks

File: documentation/MeetingMinutes/MoM-2024-03-12.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+* [Issue #85](https://github.com/camaraproject/DeviceLocation/issues/85)
+  - As requested by Joachim, issue will remain open until TEF uploads the document with more detail or the information is added to an API_documentation file to keep in the repo for future references.
+    - Joachim: Should be also aligned/synchronysed with Geofencing
+
+## AOB
+
+<p>
+
+- Next call would be on **March 26th, 2024**
+- Easter week, discuss if people would be available
```
Comment (jlurien) [inline documentation/MeetingMinutes/MoM-2024-03-12.md:136]:
```suggestion
- Easter week, but we keep the meeting
```

File: documentation/MeetingMinutes/MoM-2024-03-12.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+* [Issue #85](https://github.com/camaraproject/DeviceLocation/issues/85)
+  - As requested by Joachim, issue will remain open until TEF uploads the document with more detail or the information is added to an API_documentation file to keep in the repo for future references.
+    - Joachim: Should be also aligned/synchronysed with Geofencing
+
+## AOB
+
+<p>
+
+- Next call would be on **March 26th, 2024**
```
Comment (jlurien) [inline documentation/MeetingMinutes/MoM-2024-03-12.md:135]:
```suggestion
- Next call will be on **March 26th, 2024**

```

**Comment by jlurien** [comment]
Reviewed, you can now approve and merge, please @bigludo7

**Review by bigludo7 — APPROVED** [review]

**Pull request merged**